NEWS.md

```diff
@@ -4,6 +4,11 @@
 
 NOT RELEASED YET
 
+ * Fix MSVC builds by adding missing LIBRSYNC_EXPORT to variables in
+   librsync.h, add -DLIBRSYNC_STATIC_DEFINE to the sumset_test and
+   rdiff targets, and correctly install .dll files in the bin directory.
+   (adsun701, https://github.com/librsync/librsync/pull/161)
```

Comment:
Remove the bit about rdiff here... we want rdiff to dynamically link against the rsync library.

We only required sumset_test to be statically linked because it explicitly tests functions that are not exported by the rsync library.

Reply:
Fixed.